client/reader/search-stream/index.jsx

```diff
@@ -135,9 +187,22 @@ const FeedStream = React.createClass( {
 	},
 
 	render() {
+		var suggestions = null;
+		var lang = i18nUtils.getLocaleSlug();
+		if ( this.searchSuggestions[lang] ) {
+			suggestions = shuffle( this.searchSuggestions[lang] ).slice( 0, 3 );
+			suggestions = suggestions.map( function( query ) {
+				return <a href={ '/read/search?q=' + encodeURIComponent( query ) } >{ query }</a>;
+			} );
+			//join the link elements into a comma separated list
+			suggestions = suggestions.slice(1).reduce(function(xs, x, i) {
```

Comment:
Tried that and it didn't work because it cast the React components into strings '[Object object]' and hence I ended up at this. :/